js/util/misc.ts

```diff
@@ -25,8 +25,23 @@ export const compareStudentsByName = (student1: Map<string, any>, student2: Map<
   }
 };
 
+// A fake MD5 hash that we'll use whenever the answer field is
```

Comment:
`answer field is` "?" I'm assuming undefined as the variable is named?

Reply:
good catch